addon-templates/helm/monocular/values.yaml

```diff
@@ -0,0 +1,95 @@
+# Default values for monocular.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+api:
+  replicaCount: 2
+  image:
+    repository: gcr.io/helm-ui/monocular-api
+    tag: latest
+    pullPolicy: Always
+  service:
+    name: monocular-api
+    type: NodePort
+    externalPort: 80
+    internalPort: 8081
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  config:
+    repos:
+      # Official repositories
+      - name: stable
+        url: http://storage.googleapis.com/kubernetes-charts
+        source: https://github.com/kubernetes/charts/tree/master/stable
+      - name: incubator
+        url: http://storage.googleapis.com/kubernetes-charts-incubator
+        source: https://github.com/kubernetes/charts/tree/master/incubator
+      # Add your own repository
+      #- name: my-repo-name
+      #  url: my-repository-url
+      #  source: my-repository-source
+    # Enable Helm deployment integration
+    releasesEnabled: false
+    # Cache refresh interval in sec.
+    cacheRefreshInterval: 3600
+
+ui:
+  replicaCount: 2
+  image:
+    repository: gcr.io/helm-ui/monocular-ui
+    tag: latest
+    pullPolicy: Always
+  service:
+    name: monocular-ui
+    type: NodePort
+    externalPort: 80
+    internalPort: 80
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  # ui-config populate
+  googleAnalyticsId: UA-XXXXXX-X
+  appName: Monocular
+
+prerender:
+  replicaCount: 1
+  image:
+    repository: migmartri/prerender
+    tag: 0.1
+    pullPolicy: IfNotPresent
+  cacheEnabled: true
+  service:
+    name: prerender
+    type: NodePort
+    externalPort: 80
+    internalPort: 3000
+  resources:
+    requests:
+      cpu: 100m
+      memory: 128Mi
+
+ingress:
+  hosts:
+  # Wildcard
+  -
+  # - monocular.local
+
+  ## Ingress annotations
+  ##
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
This is required. We will need nginx ingress controller installer in a separate PR.